client/my-sites/people/people-section-nav/index.jsx

```diff
@@ -67,9 +67,9 @@ class PeopleSectionNav extends Component {
 			return false;
 		}
 
-		// Disable search for wpcom followers and viewers
+		// Disable search for wpcom followers and viewers and for invites
```

Comment:
Awkward wording here, can change it to "wpcom followers, viewers, and invites" (oxford comma [optional](https://www.youtube.com/watch?v=P_i1xk07o4g)).